.github/workflows/changed_file_trigger_examples_check.yml

```diff
@@ -0,0 +1,128 @@
+name: Test Example
+on:
+  pull_request:
+  schedule:
+    # run at 00:00 of every Sunday(singapore time) so here is UTC time Saturday 16:00
+    - cron:  '0 16 * * 6'
+
+jobs:
+# This is for changed files detect and check.
+  detect-changed-files:
+    runs-on: ubuntu-latest
+    outputs:
+      changedFiles: ${{ steps.changed-files.outputs.all_changed_files }}
+    name: Check out all files
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Get all changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          since_last_remote_commit: true  # Using this can trigger action each time a PR is submitted.
+
+  prepare-changed-file-matrix:
+    name: Prepare changed file List for checking
+    needs: detect-changed-files
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
+    - name: setup matrix
```

Comment:
This step should not run if `tj-actions/changed-files@v35` does not find any changed files. Look for its output `any_changed` in its [README](https://github.com/tj-actions/changed-files)

Reply:
If I used the 

on:
  pull_request:
    paths:
      - 'examples/**'

I think here the any_changed is not necessary? Thanks!